src/components/GameGrid/Graph.tsx

```diff
@@ -1,17 +1,19 @@
 import React from 'react'
 
 import { CartesianGrid, Legend, Line, LineChart, ResponsiveContainer, Tooltip, XAxis, YAxis } from 'recharts'
+import { AxisInterval } from 'recharts/types/util/types'
+import { CurveType } from 'recharts/types/shape/Curve'
```

Comment:
Here's a cool TS feature: if there just type, use `import type` -> anything marked as this won't be included as an import in the output bundle!
```suggestion
import type { AxisInterval } from 'recharts/types/util/types'
import type { CurveType } from 'recharts/types/shape/Curve'
```